src/gui/screen_menu_info.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file screen_menu_info.hpp
+ */
+#pragma once
+
+#include "screen_menu.hpp"
+#include "WindowMenuItems.hpp"
+#include "MItem_menus.hpp"
+
+using ScreenMenuInfo__ = ScreenMenu<EFooter::On, MI_RETURN,
+#ifdef _DEBUG
+    MI_FAIL_STAT_disabled, MI_SUPPORT_disabled,
+#endif //_DEBUG
+    MI_SYS_INFO, MI_SENSOR_INFO, MI_VERSION_INFO, MI_ODOMETER, MI_EEPROM_DIAGNOSTICS>;
+
+//cannot move it to header - 'ScreenMenuInfo' has a field 'ScreenMenuInfo::<anonymous>' whose type uses the anonymous namespace [-Wsubobject-linkage]
```

Comment:
I don't understand this comment. Is it even still valid?

Reply:
it is not, I have missed it, thanks

Reply:
fixed